salt/states/ipset.py

```diff
@@ -198,19 +199,25 @@ def present(name, entry=None, family='ipv4', **kwargs):
 
     for entry in entries:
         _entry = '{0}'.format(entry)
-        if 'comment' in kwargs:
-            _entry = '{0} comment "{1}"'.format(entry, kwargs['comment'])
+        if 'timeout' in kwargs:
+            if 'comment' in _entry:
+                _entry = '{0} timeout {1} {2} {3}'.format(entry.split()[0], kwargs['timeout'], entry.split()[1], entry.split()[2])
+            else:
+                _entry = '{0} timeout {1}'.format(entry.split()[0], kwargs['timeout'])
 
         if __salt__['ipset.check'](kwargs['set_name'],
                                    _entry,
                                    family) is True:
-            ret['result'] = True
+            if test_flag == False:
```

Comment:
Please use `if test_flag is False:` as it's a little bit faster. For True/False/None, you can use `is` for comparison.